assets/shaders/player.json

```diff
@@ -0,0 +1,14 @@
+{
+  "attributes": [
+    "vertPos",
+    "vertNor"
+  ],
+  "frag": "platform_frag.glsl",
+  "name": "player_prog",
+  "uniforms": [
+    "P",
+    "V",
+    "MV"
+  ],
+  "vert": "platform_vert.glsl"
+}
```

Comment:
can you make a separate pair of shaders for this just so that the player is a different color than the platforms